docs/flow/config-language/expressions/function_calls.md

```diff
@@ -6,3 +6,21 @@ weight: 400
 ---
 
 # Function Calls
+Function calls is one more River feature that lets users build richer
```

Comment:
Should we also mention that some function calls can fail, and if they fail, evaluation of the expression fails? (json_decode is currently the only function which can fail)